apis/metadata_validation.go

```diff
@@ -19,10 +19,21 @@ package apis
 import (
 	"fmt"
 
+	"k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/validation"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	groupName = "serving.knative.dev"
```

Comment:
I don't think this should be hardcoded here.

Reply:
See 

https://github.com/knative/pkg/blob/74c5d67ea097dd45bc66a6e2212ff70058efe005/webhook/user_info.go#L38-L62

The caller should provide the respective groupName it seems.